### DISCUSSION TITLE
Why am I getting extra padding when using the Shields Badges in MDX?

### DISCUSSION THREAD

**Discussion by kakarj** [fuma-nama/fumadocs#1090] [Q&A] [answered]
<img width="824" alt="Screenshot 2024-11-22 at 3 49 28 AM" src="https://github.com/user-attachments/assets/2cd68876-7896-45ab-a6f9-86248fff6dea">

```
[![Discord](https://img.shields.io/badge/Discord-%235865F2.svg?&logo=discord&logoColor=white)](#)
```

**Comment by fuma-nama** [comment]
p element has a default margin, you can wrap it in a div element.

**Reply by kakarj** [reply]
I used a div, but it didn't solve the issue.

<img width="439" alt="Screenshot 2024-11-22 at 5 19 45 AM" src="https://github.com/user-attachments/assets/a3dfd185-39c4-4469-bb3c-c684b7bb532f">

**Reply by fuma-nama** [reply, marked as answer]
Oops I overlooked the img tag, you can disable typography styles with not-prose, like

```mdx
<div className=“not-prose”>
[![Discord](https://img.shields.io/badge/Discord-%235865F2.svg?&logo=discord&logoColor=white)](#)
</div>
```
